src/healthkit.cs

```diff
@@ -3472,9 +3845,17 @@ interface HKLiveWorkoutBuilderDelegate
 		[Abstract]
 		[Export ("workoutBuilderDidCollectEvent:")]
 		void DidCollectEvent (HKLiveWorkoutBuilder workoutBuilder);
+
+		[Watch (9,0), NoiOS, Mac (13,0), NoMacCatalyst, NoTV]
+		[Export ("workoutBuilder:didBeginActivity:")]
+		void WorkoutBuilderDidBegin (HKLiveWorkoutBuilder workoutBuilder, HKWorkoutActivity workoutActivity);
```

Comment:
In protocols (of the *Delegate kind), we typically use the second parameter for the name:

```suggestion
		void DidBeginActivity (HKLiveWorkoutBuilder workoutBuilder, HKWorkoutActivity workoutActivity);
```